Makefile

```diff
@@ -92,7 +95,7 @@ endif
 test-dev: verify test-browser-dev test-webpack
 
 deploy:
-	webpack --bail --config build/deploy/webpack.config.js
+	make build-webpack
```

Comment:
So TIL, `$(MAKE)` creates a new instance of make and then runs that command. we don't need to do that